### ISSUE TITLE
test ofMesh::load/save rigorously

### ISSUE THREAD

**Issue by kylemcdonald** [openframeworks/openFrameworks#880]
it needs to handle all these things:
- meshes vs point clouds
- different types of meshes (depending on the triangles)
- indexed vs non-indexed geometry
- colors, normals, texcoords
- ascii vs binary
- loading from externally created/defined ply files

**Comment by micuat** [comment]
I noticed that `ofMesh::load` assumes that attributes are always ordered as `x y z r g b tx ty nx ny nz`, but it should be defined in the header (e.g., meshlab saves `x y z nx ny nz r g b`, and when the file is loaded, it will end up with strange rgb and nx at https://github.com/openframeworks/openFrameworks/blob/master/libs/openFrameworks/3d/ofMesh.cpp#L905 )

Isn't it better to use Assimp or other libraries to handle ply IO?

**Comment by tgfrerer** [comment]
Yes, this is what PR https://github.com/openframeworks/openFrameworks/pull/4664 aims to address. It's currently aimed to be merged with 0.10... 

It you needed the functionality earlier, you could test the PR locally, which would also help because it would give us a way to test it in the wild =)
